controllers/application.go

```diff
@@ -50,7 +51,7 @@ func (c *ApiController) GetApplications() {
 		}
 
 		if err != nil {
-			panic(err)
+			c.ResponseError(err.Error())
```

Comment:
This issue is not that easy like a string replacemnet. First, you need to `return` after this.

Then:

1. Fix all frontend code
2. Fix any related SDK code (can use casdoor-go-sdk as first example)

Reply:
understood, will update now

Reply:
@thanhtinhpas1  Plz make sure you have tested the web UI and casdoor-go-sdk (like with Casnode or your own app) after the changes

Reply:
Ok, sure.

Reply:
@hsluoyz  Update all changes, return immediately after response error. Also, test the application in my local.

About `casdoor-go-sdk` also, Casdoor instead of panic it returns an error, then SDK also catches it. Everything seems fine now.
Please take a look.

Reply:
@thanhtinhpas1 no frontend files are changed? This won't be correct. I have seen a lot of frontend places that accept data directly instead of Response object. Those all need to be changed

Reply:
@thanhtinhpas1 any progress on this?

Reply:
@hsluoyz Many places at frontend didn't check this case, I'm going on this today.

Reply:
@hsluoyz Please take a look, I added Web UI catch error, also testing on it and `casdoor-go-sdk`.

Reply:
Does `casdoor-go-sdk` work without modification?

Reply:
After checking, I think yes, will update now.

Reply:
@hsluoyz PR relative to in `casdoor-go-sdk` MR [MR-67](https://github.com/casdoor/casdoor-go-sdk/pull/67)

Will update the corresponding changes from all comments in this MR.